code/game/gamemodes/colonialmarines/whiskey_outpost.dm

```diff
@@ -75,7 +75,7 @@
 
 	hardcore = TRUE
 
-	votable = TRUE
+	votable = FALSE
 	vote_cycle = 25 // approx. once every 5 days, if it wins the vote
```

Comment:
```suggestion
	votable = TRUE
	vote_cycle = 75 // approx. once every 5 days, if it wins the vote
```
Alternative solution. It's intended to be available once per 5 days but you can check in discord that sometimes it once per day.

Reply:
whiskey outpost is like zombies, not all people like it but its still fun and a good bit of players do like it :(

Reply:
zombies is not a voteable gamemode. Admins would still be able to force whiskey if they want.